tests/test_end_to_end.py

```diff
@@ -204,6 +205,7 @@ def test_mention_in_simple_table(tmpdir):
     assert "Out of Table: Simple Test Page" in document
 
 
+@pytest.mark.xfail
```

Comment:
@sHermanGriffiths why is this marked xfail?

Reply:
@johndgiese because of the audio block in the all blocks page used to run the test

Reply:
I think I pushed a fix for this to main, so if you rebase on the main branch it should work now!

Reply:
I'm not allowed to force push the rebased branch history

Reply:
@johndgiese I'm not allowed to force push the rebased branch history

Reply:
Try now! Just updated perms